lib/Xud.ts

```diff
@@ -87,7 +89,7 @@ class Xud extends EventEmitter {
         initPromises.push(this.raidenClient.init());
       }
 
-      this.pool = new Pool(this.config.p2p, loggers.p2p, this.db.models);
+      this.pool = new Pool(this.config.p2p, loggers.p2p, this.db.models, minCompatibleVersion);
```

Comment:
we can pass this in `handshakeData` at `pool.init` IMO. but you can leave as is, and i'll take care of this in #756.